lib/feedbackMessages.js

```diff
@@ -0,0 +1,25 @@
+class FeedbackMessages {
+  constructor(user, surveyLink) {
+    this.user = user;
+    this.surveyLink = surveyLink;
+  }
+}
+
+class IssueFeedback extends FeedbackMessages {
+  constructor(user, surveyLink) {
+    super(user, surveyLink);
+  }
+  success(msg) {
+    return `# Step feedback for ${this.user}\n${msg}** task!\n\n_please [provide feedback](${this.surveyLink}) for this lab_`;
+  }
+
+  failure(err) {
+    return `# ${this.user} It looks like you have an error 😦\n**We expected:**\n ${err.expected}\n**We received:**\n ${err.got}`;
+  }
+
+  error(err, payload) {
+    return `# ${err.name}\n${err.userMessage}\n**payload details:**\n\`\`\`${payload.err}\`\`\``;
```

Comment:
```suggestion
    return `# Something went wrong\nIt's probably not your fault. Some additional information about the error is below. You can try doing the steps in the lab again, wait some time, or report the issue to the lab author. ${err.name}\n${err.userMessage}\n**payload details:**\n\`\`\`${payload.err}\`\`\``;
```

> report the issue to the lab author

On this point, do we want to -- or even have the ability to -- give the learner a link back to the author's repo? We may want to punt on this topic and leave it as a suggestion rather than a link, and maybe make a note of it as a potential first step in #1

Reply:
I see what you're trying to do here, but this is handled in a different part of the error handling.  The `userMessage` is what contains a similar thing to what you are trying to add.  This would only run if there is a non course related problem.

The current `userMessage` for all custom errors is `"Oops, looks like something isn't working right.  This is most likely not your fault!  Please open an issue in this lab's template repository!";`